src/frontends/lean/parser.cpp

```diff
@@ -637,7 +637,15 @@ unsigned parser_info::get_local_index(name const & n) const {
 
 void parser_info::get_include_variables(buffer<expr> & vars) const {
     m_include_vars.for_each([&](name const & n) {
-            vars.push_back(*get_local(n));
+            if (auto v = get_local(n)) {
+                vars.push_back(*v);
+            }
+        });
+}
+
+void parser_info::get_include_var_names(buffer<name> & vars) const {
+    m_include_vars.for_each([&](name const & n) {
```

Comment:
For include variables, isn't the name always the pretty name of the local constant?  (That is, this function is not necessary.)

Reply:
No because if you're in a tactic, the list of names returned will include variables that are not in scope if they are not `include`-d.

Reply:
I'm not sure I understand your answer.  Can you give an example where `get_include_variables` and `get_include_var_names` return different results?

Reply:
```lean
variables (a b c : nat)
include a
example : true := 
begin
   -- if parse a tactic argument line here, `get_include_variables` returns `[a]` and `get_include_var_names` returns `[a,b,c]`.
end
```

Reply:
Ah, that's because `get_local` returns none.  Got it!